optaplanner-core/src/test/java/org/optaplanner/core/impl/localsearch/decider/acceptor/AbstractAcceptorTest.java

```diff
@@ -25,9 +25,9 @@
 
 public abstract class AbstractAcceptorTest {
 
-    protected LocalSearchMoveScope buildMoveScope(LocalSearchStepScope stepScope, int score) {
-        LocalSearchMoveScope moveScope = new LocalSearchMoveScope(stepScope);
-        Move move = mock(Move.class);
+    protected <T> LocalSearchMoveScope<T> buildMoveScope(LocalSearchStepScope<T> stepScope, int score) {
```

Comment:
Can we replace T with Solution_ so it's clear that a solution class is expected there?